training/model/Makefile

```
@@ -0,0 +1,22 @@
+FROM ?= 
+
+REGISTRY ?= quay.io
+REGISTRY_ORG ?= ai-lab
+IMAGE_NAME ?= granite-model
```

Required change: In `quay.io/ai-lab` the granite model image is: [granite-7b-lab](https://quay.io/repository/ai-lab/granite-7b-lab)
```suggestion
IMAGE_NAME ?= granite-7b-lab
```